src/lib/Behat/PageObject/TrashPage.php

```diff
@@ -30,11 +30,17 @@ class TrashPage extends Page
      */
     public $dialog;
 
+    /**
+     * @var \EzSystems\EzPlatformAdminUi\Behat\PageElement\Tables\TrashTable
+     */
+    public $trashTable;
+
     public function __construct(UtilityContext $context)
     {
         parent::__construct($context);
         $this->route = '/admin/trash/list';
         $this->adminList = ElementFactory::createElement($this->context, AdminList::ELEMENT_NAME, 'Trash', TrashTable::ELEMENT_NAME, $this::ITEM_RESTORE_LIST_CONTAINER);
```

Comment:
IMHO adminList variable should be removed, and trashTable should implement `verifyVisiblity` that will we used in verifyElements here.